packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -189,6 +191,18 @@ function useProcessMedia({
           return;
         }
 
+        const fileInfo = await getFileInfo(file);
```

Comment:
Same question here- https://github.com/GoogleForCreators/web-stories-wp/pull/11454/files#r912080543

We have a lot of this information already in the resource.

Reply:
Let me reply at https://github.com/GoogleForCreators/web-stories-wp/pull/11454/files#r912080543